src/test/suite/format/format.test.ts

```diff
@@ -21,18 +21,40 @@ const formatTests: FormatTest[] = [
     {
         filenameRoot: '25-multiline-string',
     },
+    {
+        filenameRoot: '40-command-inside-text',
```

Comment:
Let's make these changes in the relevant PRs themselves so we know what tests map to what code :)